informers.go

```diff
@@ -49,9 +54,12 @@ func updateMachineTypes(virtCli kubecli.KubevirtClient) error {
 		machineType := vm.Spec.Template.Spec.Domain.Machine.Type
 		machineTypeSubstrings := strings.Split(machineType, "-")
 		version := machineTypeSubstrings[2]
+		if len(machineTypeSubstrings != 3) {
+			return nil
+		}
 		
 		if strings.Contains(version, "rhel") && version < "rhel9.0.0" {
```

Comment:
We need to change `version < "rhel9.0.0"`.
Basically, we need to check if the x-stream version (9 in this case) is >8. In other words, if it's less or equal to 8 (e.g. 8.4.0) it needs to be updated

Reply:
Yup, you have a point.

As long as we're sure this is indeed always the format, your approach is completely valid. Saying that, it is never too bad to add some more validations, as this can go silently wrong if something changes (what if the string is empty for some reason? or if the format changes slightly, like `rhel-8.0.0` instead of `rhel8.0.0`).

But again, if you're certain that this is always the format, feel free to leave it as it :+1: